packages/neuron-wallet/src/block-sync-renderer/sync/indexer-cache-service.ts

```diff
@@ -51,10 +49,7 @@ export default class IndexerCacheService {
       return
     }
 
-    return {
-      blockNumber: result.blockNumber.toString(),
-      blockHash: result.blockHash,
-    }
+    return result.blockNumber.toString()
```

Comment:
I removed it because https://github.com/nervosnetwork/neuron/blob/develop/packages/neuron-wallet/src/block-sync-renderer/sync/indexer-connector.ts#L64-L78 does not use the block hash.

Reply:
I see, it should be removed until we are going to handle the case of a fork.